Tests/ApolloTests/ReadFieldValueTests.swift

```diff
@@ -105,14 +119,28 @@ class ReadFieldValueTests: XCTestCase {
     XCTAssertNil(value)
   }
   
-  func testGetOptionalScalarWithWrongType() throws {
+  func testGetOptionalScalarWithDifferentType() throws {
     let object: JSONObject = ["name": 10]
     let field = GraphQLField("name", type: .scalar(String.self))
-    
+
+    XCTAssertNoThrow(try readFieldValue(field, from: object))
```

Comment:
I think this is redundant since the test `throws` - if this were to throw here or where you're getting the value, the whole test would fail